phpBB/styles/prosilver/theme/common.css

```diff
@@ -1269,6 +1269,10 @@ ul.linklist:after,
 	padding: 4px 6px;
 }
 
+.badge.hidden {
+	display: none;
+}
+
```

Comment:
Agree with both, but if the !important attribute is added, then this would be redundant